Examples/UIExplorer/js/ActivityIndicatorExample.js

```diff
@@ -150,19 +151,50 @@ exports.examples = [
       return <ToggleAnimatingActivityIndicator />;
     }
   },
-  {
-    title: 'Custom size',
-    render() {
-      return (
-        <ActivityIndicator
-          style={[styles.centering, {transform: [{scale: 1.5}]}]}
-          size="large"
-        />
-      );
-    }
-  },
 ];
 
+if (Platform.OS === 'android') {
```

Comment:
Let's keep the scale example and just add one with a custom size that is android only.